packages/liferay-npm-scripts/src/scripts/webpack.js

```diff
@@ -33,13 +31,30 @@ module.exports = function(...args) {
 				...args.slice(watch + 1)
 			];
 
-			spawn('webpack-dev-server', [
-				'--config',
-				WATCH_CONFIG_FILE,
-				...otherArgs
-			]);
+			withWebpackConfig(WEBPACK_DEV_CONFIG_FILE, configFilePath => {
+				spawnSync('webpack-dev-server', [
+					'--config',
+					configFilePath,
+					...otherArgs
+				]);
+			});
 		}
 	} else {
-		spawn('webpack', args);
+		withWebpackConfig(WEBPACK_CONFIG_FILE, configFilePath => {
+			spawnSync('webpack', ['--config', configFilePath, ...args]);
+		});
 	}
 };
+
+function withWebpackConfig(filename, callback) {
+	const mergeBabelLoaderOptionsPath = require.resolve(
```

Comment:
This should also be `path.resolve`, right? You've got a relative path and you don't need to involve the Node dependency resolution algorithm.

Reply:
Mmm, I think it should be `require.resolve`. I'm getting the absolute path of the file that corresponds to a node module so that I can then inject it in the temporary `webpack.config.js` file that will be read and loaded by webpack in a different process. 

I could also compute it with `path.join` and `__dirname` since I'm in a sibling module, but `require.resolve` looks more elegant, IMO...

Reply:
I *guess* `require.resolve` will short-circuit as soon as it sees you using a relative path anyway, so it won't invoke the entire machinery of the node resolution algorithm ie. it will be basically the same as what I was suggesting you write.